tools/experiments/experiments.js

```diff
@@ -195,20 +196,22 @@ function isExperimentOn_(id) {
  * @param {boolean=} opt_on
  */
 function toggleExperiment_(id, name, opt_on) {
+  const currentlyOn = isExperimentOn_(id);
+  const on = opt_on !== undefined ? opt_on : !currentlyOn;
```

Comment:
Double negation is a pet-peeve. How about `on = opt_on === undefined ? !currentlyOn : optOn`?